### DISCUSSION TITLE
Support discussions for private repos

### DISCUSSION THREAD

**Discussion by dwightkelly** [community/community#3355] [Discussions]
We would like to use discussions feature in our private repos. Currently it's only supported for public repos.

**Comment by PranamBhat** [comment]
Hello

In the private repos you will be the only person contributing to the repository. When it comes to more than one user, we have "Organizations" option. You can opt that one.

"**Private Repos**" meant to be with the repository owner itself. It should not mean to be shared with others. 

Have a nice day!

**Reply by dwightkelly** [reply]
No, private repo can’t be seen outside of owner and invited collaborators.

**Reply by PranamBhat** [reply]
Correct. So, problem solved!

**Reply by dwightkelly** [reply]
Smarty answers aren’t helpful. Discussions are not available for private repos in an organization....  they would be useful for our team instead of creating new issue to ask question
